FrontEnd/src/components/Contact/Contact.jsx

```diff
@@ -1,28 +1,201 @@
-import useScrollToTop from '../../hooks/useScrollToTop';
+
+const CATEGORY_OPTIONS = [
+    { value: 'Технічне питання', label: 'Технічне питання' },
+    { value: 'Рекомендації', label: 'Рекомендації' },
+    { value: 'Питання', label: 'Питання' },
+    { value: 'Інше', label: 'Інше' },
+];
 
 const Contact = () => {
-  useScrollToTop();
-
-  return (
-    <div className={styles['contact_container']}>
-      <div className={styles['contact__link_container']}>
-        <LinkContainer />
-        <img className={styles['contact__img1']}
-          src={`${process.env.REACT_APP_PUBLIC_URL}/img/dots_12x10.png`}
-          alt="dots_12x10.png" />
-      </div>
-      <div className={styles['contact__text_container']}>
-        <h2 className={styles['contact__title']}>{contactText.title}</h2>
-        {renderContent(TEXT_CONTENT)}
-      </div>
-    </div>
-  );
+    const [loading, setLoading] = useState(false);
+    const [showModal, setShowModal] = useState(false);
+    const [modalMessage, setModalMessage] = useState('');
+    const [percent, setPercent] = useState(-50);
+    const timerRef = useRef();
+
+    useEffect(() => {
+        timerRef.current = setTimeout(() => {
+            setPercent((v) => {
+                const nextPercent = v + 5;
+                return nextPercent > 150 ? -50 : nextPercent;
+            });
+        }, 100);
+        return () => clearTimeout(timerRef.current);
+    }, [percent]);
+
+    const {
+        register,
+        handleSubmit,
+        reset,
+        control,
+        formState: { errors },
+      } = useForm({
+        mode: 'all',
+        defaultValues: {
+            'message': 'Привіт, хочу повідомити...',
+            'email': '',
+            },
+      });
+
+    const onSubmit = async (value) => {
+        setLoading(true);
+        try {
+            const response = await axios.post(`${process.env.REACT_APP_BASE_API_URL}/api/admin/feedback/`, {
+                email: value.email,
+                message: value.message,
+                category: value.category,
+            });
+
+            if (response.status === 200 || response.status === 201) {
+                setModalMessage('Повідомлення успішно надіслано!');
+                reset();
+            }
+        } catch (error) {
+            setModalMessage('Щось пішло не так! Будь ласка, спробуйте ще раз');
+        } finally {
+            setShowModal(true);
+            setLoading(false);
+        }
+    };
+
+    const handleCancel = () => {
+        reset();
+    };
+
+    const closeModal = () => {
+        setShowModal(false);
+    };
+
+
+    return (
+        <div className={styles['contact_container']}>
+            <div className={styles['contact__link_container']}>
+                <LinkContainer />
+                <img
+                    className={styles['contact__img1']}
+                    src={`${process.env.REACT_APP_PUBLIC_URL}/img/dots_12x10.png`}
+                    alt="dots_12x10.png"
```

Comment:
This alt is not descriptive. Check rules to write alt for images https://abilitynet.org.uk/news-blogs/five-golden-rules-compliant-alt-text

Reply:
fixed it